### ISSUE TITLE
Android 5.1.1 Edit Media Notes

### ISSUE THREAD

**Issue by DaveWalt** [fisharebest/webtrees#699]
Samsung S6 G920F
Android 5.1.1
Default browser
KSWEB - lighttpd/MySQL/PHP5.6.10
webtrees v1.7.2
No internet connection - localhost:8080/webtrees

Editing of media notes when saved adds the corresponding ged lines to the note i.e. line number+tag+text is appended.

Fix
Tick Automatically approve changes made by this user.

**Comment by fisharebest** [comment]
> Editing of media notes when saved adds the corresponding ged lines to the note i.e. line number+tag+text is appended.

I don't follow this.

Can you post the GEDCOM data of the original record, tell me exactly how you edit it, and then post the GEDCOM data of the edited record.

**Comment by DaveWalt** [comment]
Since first posting this problem the edit issue has been fixed:
1. I have dropped lighttpd and now use nginx.
2. The developer has updated KSWEB from 3.33 to 3.4 and now 3.41:
nginx 1.7.3 to 1.9.3
PHP 5.6.10 to 5.6.12 
general improvements?

Previous webtrees behaviour was:
Edit user - auto update changes - not ticked
Individual - Album - Edit Media
1 FILE c1901 Joseph Armstrong+Family.jpg
2 FORM jpg
3 TYPE electronic
2 TITL c1901 Joseph Armstrong+Family
1 _PRIM N
1 NOTE RG13/4667/115/10 Widower Joseph Armstrong
The note appears as: RG13/4667/115/10 Widower Joseph Armstrong

A simple edit:
RG13/4667/115/10 Widower Joseph Armstrong to RG13/4667/115/10 Widower Joseph Armstrongx

Save - Accept Pending changes - The note appeared as:
RG13/4667/115/10 Widower Joseph Armstrong1 NOTE RG13/4667/115/10 Widower Joseph Armstrongx

Tick to allow user auto update changes, repeat above and the edit was displayed correctly.
